clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1,4 +1,3 @@
-//===- SemaSYCL.cpp - Semantic Analysis for SYCL constructs ---------------===//
```

Comment:
Not sure how/when I did this. I'll correct this along with other changes requested by reviewers.